lib/os/thread_posix.cpp

```diff
@@ -30,7 +30,9 @@
 #    include <pthread.h>
 
 #    ifdef __linux__
-#        include <linux/unistd.h>
+#        ifndef __MUSL__
```

Comment:
Prefer detecting this at configure time. The CMake script should check whether the include file is available. (The same applies to the other includes here, but that may be more than you want to change.)

Reply:
Good point. Fixed in the commit https://github.com/SNSystems/pstore/pull/93/commits/88195a24bcbd4e6e349f08bc63cdc1197b10f153.